website/docs/guides/using-terraform-templates.mdx

```diff
@@ -9,22 +9,24 @@ import TierLabel from "../_components/TierLabel";
 
 This guide will show you how to use a template to create a Terraform resource in Weave GitOps Enterprise.
 
+## CLI guide
+
 ### Pre-requisites
 - Install [Weave GitOps Enterprise](installation.mdx#weave-gitops-enterprise) with [TF-Controller installed](installation.mdx#tf-controller) and [TLS enabled](../configuration/tls.md).
 
-### 1. Add a template to your cluster 
+### 1. Add a template to your cluster
 
-Add a Terraform `GitOpsTemplate` manifest, such as the one below, to your config repository at `./clusters/<cluster-name>/tf-template.yaml` or any path that Flux is configured to watch. Commit and push these changes. Once a template is available in the cluster, it can be used to create a Terraform resource, which will be shown in the next step.
+Add a Terraform `GitOpsTemplate` manifest, such as the one below, to your config repository at `./clusters/<cluster-name>/tf-template.yaml`. If  or any path that Flux is configured to watch. Commit and push these changes. Once a template is available in the cluster, it can be used to create a Terraform resource, which will be shown in the next step.
```

Comment:
Can we reword `. If  or any path that Flux is configured to watch.` a bit here? It's a bit difficult to understand if you read it on the doc
 
![Screenshot 2022-07-07 at 18 24 57](https://user-images.githubusercontent.com/10995114/177834289-fbaabc51-cd42-4751-b2c0-59bede82a3c6.png)

Reply:
Good spot! I changed this and added a bunch of info so that it's clearer :)